pkgs/applications/version-management/mercurial/default.nix

```diff
@@ -55,7 +55,7 @@ let
         rustPlatform.fetchCargoVendor {
           inherit src;
           name = "mercurial-${version}";
-          hash = "sha256-k/K1BupCqnlB++2T7hJxu82yID0jG8HwLNmb2eyx29o=";
```

Comment:
Hrm, I'm getting `sha256-k/K1BupCqnlB++2T7hJxu82yID0jG8HwLNmb2eyx29o=` against staging on x86_64-linux and aarch64-linux.